migrations/sql/afterMigrate__99__assign_permissions.sql

```diff
@@ -0,0 +1,24 @@
+/**
+Allow MDS user to CRUD anything on the mds schema
+**/
+GRANT ALL PRIVILEGES ON DATABASE mds TO mds;
+GRANT ALL PRIVILEGES ON SCHEMA public TO mds;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mds;
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO mds;
+
+/**
+Allow NRIS user to CRUD anything on the nris schema
+**/
+GRANT ALL PRIVILEGES ON SCHEMA nris TO nris;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA nris TO nris;
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA nris TO nris;
+
+/**
+Allow logstash user to READ anything on the mds schema
+**/
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO logstash;
+
+/**
+Allow metabase user to READ anything on the mds schema
+**/
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO metabase;
```

Comment:
i think @itchison wants metabase to be able to cut across the schemas, so it will need to have access the NRIS schema as well.

Reply:
💯